src/Benchmarks/ReflectionPerf.cs

```diff
@@ -61,5 +62,21 @@ public object ExecuteMarshalingForCustomObject()
         {
             return instance.NewWrappedClassObject;
         }
+
+        [Benchmark]
+        public void IntEventSource()
+        {
+            ClassWithMarshalingRoutines instance = new ClassWithMarshalingRoutines();
```

Comment:
Should this be done in setup?

Reply:
I did it to create a new object each time a test is run to better simulate the real-life scenario of creating an object and then adding an event handler. A real app would not add the event handler to the same object multiple times